docs/content/users/quickstart.md

```diff
@@ -13,7 +13,7 @@ To get started with [Backdrop](https://backdropcms.org), clone the project repos
     ```bash
     mkdir my-backdrop-site && cd my-backdrop-site
     curl -LJO https://github.com/backdrop/backdrop/releases/latest/download/backdrop.zip
-    unzip ./backdrop.zip && rm backdrop.zip && mv -f ./backdrop/{.,}* . && rm -r backdrop
+    unzip ./backdrop.zip && rm -f backdrop.zip && bash -c "mv -f ./backdrop/{.,}* ." && rm -rf backdrop
```

Comment:
How about this? I just hate using fancy things (like we were using before) and running bash is fancy too. 

```suggestion
    unzip ./backdrop.zip && rm -f backdrop.zip && mv backdrop/* backdrop/.[a-z]* . && rm -rf backdrop
```

Reply:
Opened 
* https://github.com/backdrop/backdrop-issues/issues/6675

Another way around this would be to ask them to go to the parent directory and unzip, then rename `backdrop`. But that may overwrite things that already exist, etc.

Reply:
```
$ zsh
$ mv backdrop/* backdrop/.[a-z]* . && rm -rf backdrop
...
mv: cannot stat 'backdrop/.editorconfig': No such file or directory
mv: cannot stat 'backdrop/.gitignore': No such file or directory
mv: cannot stat 'backdrop/.htaccess': No such file or directory
$ echo $?
1
```

I see the same error in zsh.
I want to add, that it *really* moves these files `.editorconfig`, `.gitignore`, `.htaccess`.

The real problem is that it can't run `&& rm -rf backdrop` at the end because the return code from the previous statement is `1`.

Reply:
I think we can just move `rm -rf backdrop` to a new line.

Reply:
That's fine. Remember, of course, that `;` is the same asa a new line, if you ever need it.

> I see the same error in zsh.

Wow, I thought that was a widely provided feature, thought it was simpler than the `{}` feature

> I want to add, that it really moves these files .editorconfig, .gitignore, .htaccess.

Yes, as expected